homeassistant/components/recorder/__init__.py

```diff
@@ -135,7 +135,7 @@ async def async_setup(hass: HomeAssistant, config: ConfigType) -> bool:
     exclude_attributes_by_domain: dict[str, set[str]] = {}
     hass.data[EXCLUDE_ATTRIBUTES] = exclude_attributes_by_domain
     conf = config[DOMAIN]
-    entity_filter = convert_include_exclude_filter(conf)
+    entity_filter = convert_include_exclude_filter(conf).get_filter()
```

Comment:
We don't need the wrapped object here since we don't care if entities are specifically included or excluded so we can use the underlying filter which does not have the `__call__` overhead